components/Work/TopInfo.tsx

```diff
@@ -87,8 +87,9 @@ const WorkTopInfo: React.FC<TopInfoProps> = ({ manifest, work }) => {
       <div>
         <h2>Collection</h2>
         <Card
-          title={work.collection_title}
-          description="Collection description"
+          title={`Ima title`}
+          description="Foo bar oh my"
```

Comment:
This somehow was reverted back to the **Ima Title**

```suggestion
          title={work.collection_title}
          description="Collection description"
```

Reply:
Good catch.  Changes now updated.